Documentation/Architecture/InputSystem/CoreSystem.md

```diff
@@ -4,7 +4,8 @@ At the heart of the input system is the [MixedRealityInputSystem](https://github
 which is a service that is responsible for initializing and operating all of the input related
 functionality associated with the MRTK.
 
-> Note that it is assumed that readers have already read and have a basic understanding of the
+> [!NOTE]
```

Comment:
Does this show up differently on the docs page? It doesn't seem to resolve to anything on GitHub (I think this might be a Microsoft Docs special tag?):
![image](https://user-images.githubusercontent.com/3580640/61737712-a424db80-ad3d-11e9-80c3-8b615667142c.png)

Reply:
Yes it only shows like that in GitHub. On GitHub.io actual doc page, it looks something like this: 
![image](https://user-images.githubusercontent.com/25975362/61738803-f0711b00-ad3f-11e9-8ddb-32ab37d03ce6.png)